R/survival_qp.R

```diff
@@ -66,8 +66,8 @@ survival_qp <- function(B_X, trt, times, events, t, lambda = 0, lowlim = 1, upli
   # imbalance_root <- t(w) %*% B - Bbar_X
   # imbalance <- imbalance_root %*% t(imbalance_root)
   
-  imbalance <- sum(((t(weights) %*% B_X) - Bbar_X)^2) 
-    
+  imbalance <- mean(((t(weights) %*% B_X) - Bbar_X)^2) 
+  
```

Comment:
Can this be separated out into the vector of imbalances for treated and control units? Then we can check whether this is working by checking whether the imbalance is small.